draft-ietf-quic-transport.md

```diff
@@ -3688,11 +3694,13 @@ to adjust for any intentional delays, which is important for getting a better
 estimate of the path RTT when acknowledgments are delayed.  A packet might be
 held in the OS kernel or elsewhere on the host before being processed.  An
 endpoint MUST NOT include delays that it does not control when populating the
-ACK Delay field in an ACK frame.
+ACK Delay field in an ACK frame, but endpoints SHOULD include buffering delays
+caused by unavailability of decryption keys.
```

Comment:
Do we need to explain why this is a SHOULD? We could say that if a packet containing CRYPTO data from the previous PN was lost, the buffering delay will be > 1 RTT. That should be a very good motivation to follow this SHOULD.